pymedphys/_experimental/pinnacle/rtdose.py

```diff
@@ -415,6 +427,11 @@ def convert_dose(plan, export_path):
     pixel_binary_block = struct.pack("%sh" % len(pixelvaluelist), *pixelvaluelist)
     ds.PixelData = pixel_binary_block
 
+    # If Feet first, flip the dose grid
+    if patient_position in ("FFS", "FFP"):
+        arr = ds.pixel_array
+        ds.PixelData = np.flip(arr, axis=0).tostring()
```

Comment:
It seems like we should have a function that is within PyMedPhys that converts pixel data from one orientation (say 'HFS')  to another (for example 'FFS'):

```python
supported_orientations = ('HFS', ... etc)  # maybe even image_orientation_map.keys() ?

def convert_pixel_orientation(pixel_data, from_orientation, to_orientation):
    if not set((from_orientation, to_orientation)).issubset(supported_orientations):
        raise ValueError("Provided orientation not supported.")
	...
	do_stuff()    
    ...

	return transformed_pixel_data
```

Does that seem doable? And then that function could be called here.

Reply:
A function like that would happily reside in a place such as a new file at `pymedphys/_dicom/pixelarray.py`

Reply:
Hi @SimonBiggs , I'm not sure how to do this to be honest. I think a general function for this would need to adjust the axis in all directions to convert between the orientations. But for some reason the Pinnacle dose only needs to be flipped to convert between Head first and feet first, the other orientations seem to already be accounted for.

Any ideas why this might be? Am I missing something obvious? If not perhaps we'll leave it for now? Since this is probably only Pinnacle specific there wouldn't be much benefit in pulling out a general function.

Reply:
> Any ideas why this might be? Am I missing something obvious? If not perhaps we'll leave it for now? Since this is probably only Pinnacle specific there wouldn't be much benefit in pulling out a general function.

Hmm, not sure why that is. I agree, let's leave it for now.